### PR TITLE
K8s storage links

### PR DESCRIPTION
**Pull request by hunchback**
this PR automates the k8s storage scenario as found here: https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/ with the following:

```
cd tests/k8s
./storage.sh start
```

this scenario creates:
- create pv/task-pv-volume
- create pvc/task-pv-claim
- create pod/task-pv-pod
- reference sc/standard

the created links:
- task-pv-pod --> task-pv-claim
- task-pv-claim --> task-pv-volume (sometimes missing due to backend race-condition)
- standard --> task-pv-claim (sometimes missing due to backend race-condition)
- standard --> task-pv-volume

The above race-condition, causing missing links appears to come from issue in graph/linker code (is beyond the scope of this current PR - and will be addressed separately).

### CODE REVIEW

**Comment by hunchback** [comment]
run skydive-functional-tests-backend-orientdb

File: vendor/vendor.json [hunk 1]
```diff
@@ -2342,7 +2342,7 @@
 			"revision": "604eaf189ee867d8c147fafc28def2394e878d25"
 		},
 		{
-			"checksumSHA1": "rFSw9cpg4vxtZD7dlOtKx25h4kI=",
+			"checksumSHA1": "2jmj7l5rSw0yVb/vlWAYkK/YBwk=",
```
Comment (nplanel) [inline vendor/vendor.json]: please update your local govendor

File: Makefile [hunk 2]
```diff
@@ -309,6 +312,9 @@ statics/bindata.go: .typescript ebpf.build $(shell find statics -type f \( ! -in
 	$(call VENDOR_RUN,${GO_BINDATA_GITHUB}) go-bindata ${GO_BINDATA_FLAGS} -nometadata -o statics/bindata.go -pkg=statics -ignore=bindata.go $(BINDATA_DIRS)
 	gofmt -w -s statics/bindata.go
 
+.PHONY: compile.debug
+compile.debug: debug.flags compile
```
Comment (nplanel) [inline Makefile]: compile.debug rule is different from debug :
In case of compile.debug you bypass all the generating rules/files ( govendor genlocalfiles dpdk.build contrib )

So please keep on one way/entry point to build in debug
compile.debug will make confusion and for example make GO_BINDATA_FLAGS useless as genlocalfiles rule is not called
Please revert these changes

File: vendor/vendor.json (hunk 1 above)
Comment (hunchback) [reply vendor/vendor.json]: will remove - thanbks

File: Makefile (hunk 2 above)
Comment (hunchback) [reply Makefile]: ok

File: tests/k8s/storage.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (hunchback) [reply tests/k8s/storage.sh]:
```
./storage.sh
```

File: tests/k8s/storage.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+#
+# this is driver to running the scenario found here:
+# https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/
+
+SRC=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+: ${NAMESPACE:=default}
+echo "export NAMESPACE=$NAMESPACE"
+
+DATA=/mnt/data
```
[the remark this answers is not in the document]
Comment (hunchback) [reply tests/k8s/storage.sh]:
```
DATA=/mnt/skydive-test-storage
```

File: tests/k8s/storage.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (lebauce) [reply tests/k8s/storage.sh]: @hunchback I think that, what @nplanel meant is that this script doesn't seem to be used anywhere

[the remark this answers is not in the document]
Comment (hunchback) [reply tests/k8s/storage.sh]: I have add a new test testK8sStorageScenario which makes use of this script.

**Comment by lebauce** [comment]
@hunchback Please rebase against master

**Comment by hunchback** [comment]
run skydive-cdd-overview-tests

**Comment by hunchback** [comment]
Any additional comments?

**Comment by lebauce** [comment]
@hunchback Sorry, we are still waiting for our hardware CI to be up again so that we can run the Kubernetes tests.

**Comment by hunchback** [comment]
@lebauce - thanks for the update ...

**Comment by hunchback** [comment]
@lebauce - any updates ?

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-k8s-tests
